src/olympia/files/models.py

```diff
@@ -377,20 +377,45 @@ def get_localepicker(self):
         statsd.timing('files.extract.localepicker', (end * 1000))
         return res
 
-    @amo.cached_property
+    @property
     def webext_permissions(self):
+        """Return permissions with descriptions in defined order:
```

Comment:
We could, but even if we _never_ change the order we want them displayed in, there will be permissions that are unknown at upload time, and then become known, so they'd be in the wrong place in the list.  
The list would still have to be iterated over to treat the permissions differently anyway (e.g. known permission get a description; the all-url permissions get de-duped; etc)